ciao-controller/legacy_api.go

```diff
@@ -270,43 +281,43 @@ func listServerDetailsFlavors(c *controller, w http.ResponseWriter, r *http.Requ
 
 func legacyComputeRoutes(ctl *controller, r *mux.Router) *mux.Router {
 	r.Handle("/v2.1/{tenant}/servers/action",
-		legacyAPIHandler{ctl, tenantServersAction}).Methods("POST")
+		legacyAPIHandler{ctl, tenantServersAction, false}).Methods("POST")
 
 	r.Handle("/v2.1/flavors/{flavor}/servers/detail",
-		legacyAPIHandler{ctl, listServerDetailsFlavors}).Methods("GET")
+		legacyAPIHandler{ctl, listServerDetailsFlavors, true}).Methods("GET")
 
 	r.Handle("/v2.1/{tenant}/resources",
-		legacyAPIHandler{ctl, listTenantResources}).Methods("GET")
+		legacyAPIHandler{ctl, listTenantResources, false}).Methods("GET")
 
 	r.Handle("/v2.1/{tenant}/quotas",
-		legacyAPIHandler{ctl, listTenantQuotas}).Methods("GET")
+		legacyAPIHandler{ctl, listTenantQuotas, false}).Methods("GET")
 
 	r.Handle("/v2.1/tenants",
-		legacyAPIHandler{ctl, legacyListTenants}).Methods("GET")
+		legacyAPIHandler{ctl, legacyListTenants, true}).Methods("GET")
 
 	r.Handle("/v2.1/nodes",
-		legacyAPIHandler{ctl, legacyListNodes}).Methods("GET")
+		legacyAPIHandler{ctl, legacyListNodes, true}).Methods("GET")
 	r.Handle("/v2.1/nodes/summary",
-		legacyAPIHandler{ctl, legacyNodesSummary}).Methods("GET")
+		legacyAPIHandler{ctl, legacyNodesSummary, true}).Methods("GET")
 	r.Handle("/v2.1/nodes/{node}/servers/detail",
-		legacyAPIHandler{ctl, legacyListNodeServers}).Methods("GET")
+		legacyAPIHandler{ctl, legacyListNodeServers, true}).Methods("GET")
 
 	r.Handle("/v2.1/cncis",
-		legacyAPIHandler{ctl, legacyListCNCIs}).Methods("GET")
+		legacyAPIHandler{ctl, legacyListCNCIs, true}).Methods("GET")
 	r.Handle("/v2.1/cncis/{cnci}/detail",
-		legacyAPIHandler{ctl, legacyListCNCIDetails}).Methods("GET")
+		legacyAPIHandler{ctl, legacyListCNCIDetails, true}).Methods("GET")
 
 	r.Handle("/v2.1/events",
-		legacyAPIHandler{ctl, legacyListEvents}).Methods("GET")
+		legacyAPIHandler{ctl, legacyListEvents, true}).Methods("GET")
```

Comment:
This one is a problem with verify.sh - unless we are admin, we shouldn't use this api, we should use the per tenant api.